src/components/patch/PatchContainer.tsx

```diff
@@ -39,7 +43,15 @@ export class PatchContainer extends React.Component<Props, State> {
 
     return (
       <div>
-        <div className="patch-search-bar">Insert patch search bar here</div>
+        <div className="search-container">
+          <Paper>
+            <InputBase startAdornment={<IconButton><SearchIcon /></IconButton>}
+              className="search-input" fullWidth={true}
+              placeholder="Search Patch Descriptions..."
```

Comment:
Nit: I think the label shouldn't have the ellipses

Reply:
Fixed